core/trino-main/src/main/java/io/trino/server/ui/AuthPreviewResource.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.server.ui;
+
+import jakarta.ws.rs.container.ContainerRequestContext;
+import jakarta.ws.rs.core.Context;
+import jakarta.ws.rs.core.SecurityContext;
+
+public interface AuthPreviewResource
+{
+    AuthInfo getAuthInfo(ContainerRequestContext request, @Context SecurityContext securityContext);
```

Comment:
I don't think that this interface is needed unless we move

```
@ResourceSecurity(WEB_UI)
    @GET
    @Path(UI_PREVIEW_AUTH_INFO)
    @Produces(APPLICATION_JSON)
```
to it

Reply:
deleted this interface because it doesn't like `ContainerRequestContext` being passed as parameter when jaxrs annotations moved to the interface.